spec/features/authenticate/sign_in_spec.rb

```diff
@@ -1,13 +1,23 @@
 require "rails_helper"
 
 feature "Sign In" do
+  let!(:company) { create(:company, name: "School 13") }
+
   before do
-    create(:user, email: "example@gmail.com", password: "123456", first_name: "Bilbo", last_name: "Baggins")
+    create(
+      :user,
+      company:,
```

Comment:
I don't really know why rubocop wanna see it like here 🤔

Reply:
https://docs.rubocop.org/rubocop/cops_style.html#enforcedshorthandsyntax-always-default